src/web/client/dal/fileSystemProvider.ts

```diff
@@ -198,9 +198,6 @@ export class PortalsFS implements vscode.FileSystemProvider {
                     title: vscode.l10n.t("Saving your file ..."),
                 },
                 async () => {
-                    WebExtensionContext.telemetry.sendInfoTelemetry(
```

Comment:
Do not remove this telemetry point - i see you have added it downstream - but what if there is a failure before this call is triggered - we will loose insight on failed save scenarios.